src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1509,9 +1509,15 @@
       "additionalProperties": false,
       "properties": {
 
-        "physicalLocation": {
+        "locations": {
           "description": "The artifact and region relevant to this notification.",
```

Comment:
>artifact and region [](start = 30, length = 19)

"artifact and region" => "locations"

Reply:
<facepalm>

---
In reply to: [278289845](https://github.com/Microsoft/sarif-sdk/pull/1456#discussion_r278289845) [](ancestors = 278289845)